tensorbay/opendataset/VGGFace2/loader.py

```diff
@@ -0,0 +1,221 @@
+#!/usr/bin/env python3
+#
+# Copyright 2021 Graviti. Licensed under MIT License.
+#
+# pylint: disable=invalid-name
+
+"""Dataloaders of VGGFace2 dataset."""
+
+import csv
+import os
+from collections import defaultdict
+from itertools import islice
+from typing import Any, DefaultDict, Dict, Generator, List, Tuple, Union
+
+from tensorbay.dataset import Data, Dataset
+from tensorbay.label import Catalog, Classification, LabeledBox2D, LabeledKeypoints2D
+from tensorbay.opendataset._utility import glob
+from tensorbay.utility import chunked
+
+_DATASET_NAME = "VGGFace2"
+_SEGMENTS = ("train", "test")
+
+
+_CLASS_ATTRS = ("Sample_num", "Flag", "Gender")
```

Comment:
We use `attribute` to represent the annotation attribute in dataset.
We use `attr` to represent the attribute in python language.
So here we should use `attribute` rather than `attr` here.